test/e2e/network/dns_scale_records.go

```diff
@@ -50,10 +50,15 @@ var _ = SIGDescribe("[Feature:PerformanceDNS]", func() {
 
 	// answers dns for service - creates the maximum number of services, and then check dns record for one
 	It("Should answer DNS query for maximum number of services per cluster", func() {
-		services := generateServicesInNamespace(f.Namespace.Name, maxServicesPerCluster)
+
+		ns1, _ := f.CreateNamespace(f.BaseName, nil)
+		ns2, _ := f.CreateNamespace(f.BaseName, nil)
+
+		services := generateServicesInNamespaces([]string{ns1.Name, ns2.Name}, maxServicesPerCluster/2)
```

Comment:
Instead of just introducing this maginc constant "2", can you please introduce a constant:
"maxServicesPerNamespace" (for now set it to 5000)

and then:
- create necessary number of namesapces (maxServicesPerCluster + maxServicesPerNamespace - 1 / maxServicesPerNamespace)
- in each of namespaces create necessary number of services?